example/object_detection_ssd_mobilenet/assets/model/lite-labelmap.txt

```diff
@@ -1,90 +0,0 @@
-person
```

Comment:
Nice. So I know the image classification model has metadata on it to include the labels - did it work with this example?

Reply:
Yes!

![image](https://github.com/tensorflow/flutter-tflite/assets/86299638/bf86c80d-61ab-43c5-9643-3a537a8f8041)